server.js

```diff
@@ -533,3 +533,17 @@ setInterval(function() {
                 "Warning! All builders are idle, but there are " + queued + " PRs in the queue.");
     });
 }, THIRTY_MINUTES);
+
+setInterval(function() {
+    homu.retrieveSlaves(function(slaves) {
+		for(var slaveName in slaves){
```

Comment:
Could we avoid using tabs here?

Reply:
Sorry! Must have left vim on paste mode. I'll fix that.